python-checks/src/main/java/org/sonar/python/checks/tests/SkippedTestNoReasonCheck.java

```diff
@@ -38,19 +41,23 @@ public class SkippedTestNoReasonCheck extends PythonSubscriptionCheck {
 
   private static final String MESSAGE = "Provide a reason for skipping this test.";
 
+  private static final Set<String> skipDecoratorsFQN = new HashSet<>(List.of("unittest.case.skip", "pytest.mark.skip"));
+  private static final Set<String> skipCallExpressionsFQN = new HashSet<>(List.of("pytest.skip"));
+
   @Override
   public void initialize(Context context) {
     context.registerSyntaxNodeConsumer(Tree.Kind.DECORATOR, ctx -> {
       Decorator decorator = (Decorator) ctx.syntaxNode();
-
-      checkDecoratorSkipWithoutReason(ctx, decorator, "unittest.case.skip");
-      checkDecoratorSkipWithoutReason(ctx, decorator, "pytest.mark.skip");
+      for (String skipDecoratorFQN : skipDecoratorsFQN) {
+        checkDecoratorSkipWithoutReason(ctx, decorator, skipDecoratorFQN);
+      }
     });
 
     context.registerSyntaxNodeConsumer(Tree.Kind.CALL_EXPR, ctx -> {
       CallExpression callExpression = (CallExpression) ctx.syntaxNode();
-
-      checkCallExpressionSkipWithNoOrEmptyReason(ctx, callExpression, "pytest.skip");
+      for (String skipCallExpressionFQN : skipCallExpressionsFQN) {
```

Comment:
Same here, we shouldn't have to iterate over those names (even if there is only one 😄 ).